google/cloud/storage/benchmarks/benchmark_utils.h

```diff
@@ -116,6 +116,20 @@ std::string FormatBandwidthGbPerSecond(
   return std::move(os).str();
 }
 
+template <typename Rep, typename Period>
+std::string FormatBandwidthMiBs(std::uintmax_t bytes,
```

Comment:
Observation:  It is a little bit of a shame that this one uses "s" while the previous one uses "PerSecond".

Reply:
Fixed, maybe?